src/assets/toolkit/images/icons/chevron-down-circle-filled.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25"><circle class="sprk-SvgChevronCircleFilled--circle" cx="12.46" cy="12.54" r="11.2"/><polyline class="sprk-SvgChevronCircleFilled--arrow" points="18.36 10.64 12.46 16.54 6.56 10.64"/></svg>
```

Comment:
@afebbraro @RCopeland is this why we are getting mixed content warnings? I can't see anything else that would cause it. I wouldn't expect this to cause it either, but I dunno.

Reply:
i _think_ its from a placeholder image on the links page
https://github.com/sparkdesignsystem/spark-design-system/pull/527

Reply:
I commented here because the email directed me to a netlify log that pointed back to this PR, not that links page. https://app.netlify.com/sites/spark-design-system/deploys/5b9ab1db1f12b742318dac61